spec/filters/grok_spec.rb

```diff
@@ -665,6 +666,7 @@
 
     after do
       @file.unlink
+      FileUtils.rm_rf(pattern_dir)
```

Comment:
I guess we should only remove it if we know that it was created during the test?